leakcanary-android/src/main/java/com/squareup/leakcanary/ActivityRefWatcher.java

```diff
@@ -15,24 +15,22 @@
  */
 package com.squareup.leakcanary;
 
-import android.annotation.TargetApi;
 import android.app.Activity;
 import android.app.Application;
 import android.os.Bundle;
 
-import static android.os.Build.VERSION.SDK_INT;
-import static android.os.Build.VERSION_CODES.ICE_CREAM_SANDWICH;
 import static com.squareup.leakcanary.Preconditions.checkNotNull;
 
-@TargetApi(ICE_CREAM_SANDWICH) public final class ActivityRefWatcher {
+public final class ActivityRefWatcher {
 
+  /** @deprecated Use {@link #install(Application, RefWatcher)}. */
+  @Deprecated
```

Comment:
As a developer tool we don't strictly follow semver already. Perhaps the scope of changes will necessitate a 2.0 for other reasons though.

Also I plan to remove this method and the one I just added in a follow-up, but wanted to minimize changes here.